packages/opentelemetry-core/test/trace/tracestate.test.ts

```diff
@@ -25,30 +25,30 @@ describe('TraceState', () => {
     });
 
     it('must replace keys and move them to the front', () => {
```

Comment:
I believe we should update the description and mention that, "creates a new tracestate and replace..." WDYT?

Reply:
done